packages/rocketchat-ui-flextab/client/tabs/userInfo.js

```diff
@@ -0,0 +1,526 @@
+/* globals RoomRoles, UserRoles*/
+import moment from 'moment';
+import toastr from 'toastr';
+
+Template.userInfo.helpers({
+	name() {
+		const user = Template.instance().user.get();
+		return user.name ? user.name : TAPi18n.__('Unnamed');
+	},
+
+	username() {
+		const user = Template.instance().user.get();
+		return user.username;
+	},
+
+	email() {
+		const user = Template.instance().user.get();
+		if (user.emails && user.emails[0] && user.emails[0].address) {
+			return user.emails[0].address;
+		}
+	},
+
+	utc() {
+		const user = Template.instance().user.get();
+		if (user.utcOffset != null) {
```

Comment:
@marceloschmidt we are not requiring `===` when comparing with null https://github.com/RocketChat/Rocket.Chat/blob/develop/.eslintrc#L70